tests/e2e/runtime.go

```diff
@@ -129,15 +129,15 @@ func (sc *runtimeScenario) Fixture() (*oasis.NetworkFixture, error) {
 				Executor: registry.ExecutorParameters{
 					GroupSize:       2,
 					GroupBackupSize: 1,
-					RoundTimeout:    20,
-					MaxMessages:     128,
+					RoundTimeout:    30,
+					MaxMessages:     256,
 				},
 				TxnScheduler: registry.TxnSchedulerParameters{
 					Algorithm:         registry.TxnSchedulerSimple,
-					MaxBatchSize:      1,
-					MaxBatchSizeBytes: 1024,
-					BatchFlushTimeout: 1 * time.Second,
-					ProposerTimeout:   20,
+					MaxBatchSize:      10,
+					MaxBatchSizeBytes: 16 * 1024 * 1024, // 16 MB.
+					BatchFlushTimeout: 20 * time.Second,
```

Comment:
Why did you increase the flush timeout, this can delay tests? I would leave it as 1 second.

Reply:
It seemed a bit flaky with shorter timeouts, I'll try to change this back to 1s.

Reply:
OK, it seems to work fine with 1s too :man_shrugging: